.github/workflows/test-on-droplet.yml

```diff
@@ -0,0 +1,74 @@
+name: "Run tests on DigitalOcean Droplet"
+on:
+  push
+
+jobs:
+  build_deb:
+    name: "Run in DigitalOcean Droplet"
+    runs-on: ubuntu-latest
+    concurrency: droplet-aleph-vm
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Git unshallow
+        run: |
+          git fetch --prune --unshallow
+          git describe --tags
+
+      - name: Install doctl
+        uses: digitalocean/action-doctl@v2
+        with:
+          token: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
+
+      - name: Setup SSH private key
+        run: |
+          mkdir ~/.ssh
+          echo $DIGITALOCEAN_SSH_PRIVATE_KEY | base64 --decode > ~/.ssh/id_ed25519
+          chmod 0700 ~/.ssh
+          chmod 0600 ~/.ssh/id_ed25519
+        env:
+          DIGITALOCEAN_SSH_PRIVATE_KEY: ${{ secrets.DIGITALOCEAN_SSH_PRIVATE_KEY }}
+
+      - name: Create the Droplet
+        run: |
+          doctl compute droplet create --image debian-11-x64 \
+          --size s-1vcpu-1gb --region ams3 aleph-vm-ci \
+          --ssh-keys 18:09:36:58:79:44:bb:84:45:c8:6f:9a:f6:b8:0a:c5
+
+      - name: Build Debian Package
+        run: |
+          cd packaging && make all-podman-debian-11 && cd ..
+          ls packaging/target
+
+#      - name: Wait for the system to setup and boot
+#        run: sleep 60
+# Disabled since the build takes approximately as much time
```

Comment:
Anyway, it would be better to have a script that checks for connectivity on the VM, for example by verifying the VM is reachable every X seconds. Out of context for this PR, just saying.

Reply:
Thanks for the remark, updated.